Directory.Build.props

```diff
@@ -95,7 +95,7 @@
     <AzureSecurityKeyVaultCertificatesVersion>4.6.0</AzureSecurityKeyVaultCertificatesVersion>
     <MicrosoftGraphVersion>4.36.0</MicrosoftGraphVersion>
     <MicrosoftGraphBetaVersion>4.57.0-preview</MicrosoftGraphBetaVersion>
-    <MicrosoftIdentityAbstractionsVersion>9.0.0</MicrosoftIdentityAbstractionsVersion>
+    <MicrosoftIdentityAbstractionsVersion>9.1.1</MicrosoftIdentityAbstractionsVersion>
```

Comment:
Requires https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/pull/178 to be merged